python/schedule.hpp

```diff
@@ -78,9 +78,9 @@ struct explicit_schedule_shim: schedule_shim_base {
 struct poisson_schedule_shim: schedule_shim_base {
     using rng_type = std::mt19937_64;
 
-    arb::time_type tstart = arb::terminal_time;
-    arb::time_type freq = 10.;  // kHz
-    rng_type::result_type seed = 0;
+    arb::time_type tstart; // ms
+    arb::time_type freq; // kHz
+    rng_type::result_type seed;
 
     poisson_schedule_shim() = default;
```

Comment:
Without in-line defaults, this constructor will do whatevs, and initialize the object in an undetermined fashion. Remove this ctor too?

Reply:
yes!